### DISCUSSION TITLE
Birds Eye Default

### DISCUSSION THREAD

**Discussion by MrCaspan** [blakeblackshear/frigate#15033] [Ask A Question] [answered]
Question, just an opinion and take it only as this. Would it not make more sense to have features like `BirdsEye` or `mqtt` defaults to be `enabeled: false`? This would make for a cleaner config file that again IMO should only list things that you use or configure to be used.

Again just some feeback

**Comment by hawkeye217** [comment, marked as answer]
We've found that a large number of users will use Frigate with Home Assistant, where MQTT is required to be enabled. Also, up until Frigate 0.14, Birdseye was the primary way users would have a live view their camera streams. During the beta period for 0.14, many users asked about birdseye and wanted to continue using it. It doesn't make sense to disable it by default at least for now.

**Reply by MrCaspan** [reply]
Awesome , seems like a reason again just feedback. I'm a config cleaner, I like it being clean :)  

There are so many settings that I had no idea that I was using wrong or that I set and I didn't have to. I know some people would rather be explicit cuz that makes reading it a lot more easier than trying to figure out what the default is if the values not there. but I found for  things like the detect height and width, I had it set to 640 by 480 because at the time that was the resolution of my three cameras. then I got three more cameras whose aspect ratios were a little bit different. I had no idea that I was using CPU even though it might be small to resize those frames from that set resolution at the top of my config. then in my head I thought what do I set as the default at the top of my config and then override on the cameras. so some cameras would have a detect setting some wouldn't so it would make more sense to me to put a detect setting on each camera so that you know why Somerset and some aren't. Then I looked at the default config and I see that if you specify nothing for the height and width it defaults to whatever the frame size it comes in as akes more sense. Things like this are just really great improvements to the application to make the configuration cleaner but also easier to read for me so I'm not sure if these changes have been made since 0.14 but these are great defaults! keep up the great work guys

**Reply by hawkeye217** [reply]
Appreciate the feedback!

**Comment by MattCrBoom** [comment]
ndcn$
